setup.cfg

```diff
@@ -2,7 +2,7 @@
 universal = 1
 
 [flake8]
-ignore =
+ignore = W503, W504
```

Comment:
why u r ignoring it? You should fix the problem, not to ignore it. 
@acheshkov Do u agree that we are not allowed to ignore them?